R/tcplhit2_core.R

```diff
@@ -66,7 +69,7 @@
 #'   }
 #' @export
 #'
-tcplhit2_core <- function(params, conc, resp, cutoff, onesd,bmr_scale = 1.349, bmed = 0, conthits = TRUE, aicc = FALSE, identifiers = NULL, bmd_low_bnd = NULL, bmd_up_bnd = NULL) {
+tcplhit2_core <- function(params, conc, resp, cutoff, onesd,bmr_scale = 1.349, bmed = 0, conthits = TRUE, aicc = FALSE, identifiers = NULL, bmd_low_bnd = NULL, bmd_up_bnd = NULL, errfun = "dt4") {
```

Comment:
We should just make the errfun part of the 'params' input.  This should be done by adding the 'errfun' to the output list in the returned fitting object from tcplfit2 (similar to the model names list element).
Rationale: If the users chooses to use a particular error distribution, then this should be the error distribution used for the p2 calculation.  This software design choice prevent users from mistakenly mixing error distributions.